ui/desktop/app/templates/scopes/scope/projects/sessions/index.hbs

```diff
@@ -115,80 +115,83 @@
     </Rose::Toolbar>
 
     {{#if this.sorted}}
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            <row.headerCell>{{t 'resources.target.title'}}</row.headerCell>
-            <row.headerCell>{{t 'form.proxy.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.started.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.status.label'}}</row.headerCell>
-            <row.headerCell />
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each this.sorted as |model|}}
-            <body.row as |row|>
-              <row.headerCell class='nowrap'>
-                <Rose::Icon
-                  class={{concat 'session-status-' model.status}}
-                  @name='flight-icons/svg/entry-point-16'
-                  @size='medium'
-                />
+      <Hds::Table
+        @model={{this.sorted}}
+        @columns={{array
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'resources.target.title'))
+          (hash label=(t 'form.proxy.label'))
+          (hash label=(t 'form.started.label'))
+          (hash label=(t 'form.status.label'))
+          (hash label='')
+        }}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if (can 'read model' B.data)}}
+                {{#if B.data.isActive}}
+                  <Hds::Badge
+                    @text={{t 'resources.session.title'}}
+                    @icon='entry-point'
+                    @isIconOnly={{true}}
+                    @color='success'
+                    class='session-status-active'
```

Comment:
Is this class still needed? It looks like it only sets the color which is handled by the new `@color` field

Reply:
Good catch, I forgot to comment on this class. We have a test that uses that class to verify there is an active session. Looking at the designs for this table, it is going to change a bit and that icon and test will go away. To fight off scope creep, I told Natasha to add the class for now until we refactor the table items for the post connection workflow updates.

Reply:
We should add a TODO then to note this